### DISCUSSION TITLE
Start Menu covered by Task bar

### DISCUSSION THREAD

**Discussion by CardboardFreedom** [valinet/ExplorerPatcher#2654] [General]
When the taskbar is on the left, has auto-hide enabled, and Start Menu Position is set to At Screen Edge, the start menu will be positioned behind the task bar. The only work around for this is setting the Start Menu Position to Center, which does not have this issue but looks odd having the start menu in the center.

![Issue](https://github.com/valinet/ExplorerPatcher/assets/83144880/547c551d-0029-48a9-b93f-74a335824405)

**Comment by pyrates999** [comment]
Perhaps this is happening because Microsoft never intended you to have the windows 11 taskbar on the left or right side.
